assertions/serverIsReachable.ts

```diff
@@ -1,7 +1,49 @@
 import * as isReachable from "is-reachable";
 
 export class ServerIsReachable {
-    public static run(port: number, path: string): Promise<boolean> {
-        return isReachable("http://localhost:" + port + "/" + path);
+    public static async run(parameters, callback): Promise<void> {
+        let port = this.getValue(parameters, 'port', undefined);
+        let path = this.getValue(parameters, 'path', "");
+        let interval = this.getValue(parameters, 'intervall', 5);
+        let startupTime = this.getValue(parameters, 'startupTime', 600);
```

Comment:
10 minutes is the default value for timeout if in the tutorial no value is given. I think it would be best to give the tutorial creator the most responsibility. In some cases it might be useful to fail fast.

Reply:
And i think the possible parameters should be documented somewhere, but i'm not quite sure where.

Reply:
You could add a new file for the assertions in the documentation folder, where we list each assertion and its parameters